README.md

```diff
@@ -4,22 +4,22 @@
 [![.github/workflows/packages.yml](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/packages.yml/badge.svg)](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/packages.yml)
 [![Mergin Plugin Tests](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/run-test.yml/badge.svg)](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/run-test.yml)
 
-QGIS plugin aims to simplify handling of [Mergin Maps](https://public.cloudmergin.com/) projects.
+QGIS plugin aims to simplify handling of [Mergin Maps](https://app.merginmaps.com/) projects.
 
-[Mergin Maps](https://public.cloudmergin.com/) is a service to store, synchronise and manage GIS data and QGIS projects. Currently, [Mergin Maps](https://public.cloudmergin.com/) is used as the back-end of [Mergin Maps Input](https://inputapp.io/), an [open source](https://github.com/lutraconsulting/input) mobile application to collect data in field.
+[Mergin Maps](https://app.merginmaps.com/) is a service to store, synchronise and manage GIS data and QGIS projects. Currently, [Mergin Maps](https://app.merginmaps.com/) is used as the back-end of [Mergin Maps Input](https://inputapp.io/), an [open source](https://github.com/lutraconsulting/input) mobile application to collect data in field.
 
 <div><img align="left" width="45" height="45" src="https://raw.githubusercontent.com/MerginMaps/docs/main/src/.vuepress/public/slack.svg"><a href="https://merginmaps.com/community/join">Join our community chat</a><br/>and ask questions!</div>
 
 ## Quick start
 
 A typical workflow for setting up a project for field survey:
 1. User sets up the project in QGIS desktop
-2. The project gets uploaded to [Mergin Maps](https://public.cloudmergin.com/)
+2. The project gets uploaded to [Mergin Maps](https://app.merginmaps.com/)
 3. Projects and data will be available on [Mergin Maps Input](https://inputapp.io/) mobile application
-4. Collected data from [Mergin Maps Input](https://inputapp.io/) will be synchronised back to [Mergin Maps](https://public.cloudmergin.com/)
+4. Collected data from [Mergin Maps Input](https://inputapp.io/) will be synchronised back to [Mergin Maps](https://app.merginmaps.com/)
 5. Data will be available to view on QGIS desktop
 
 Mergin Maps QGIS plugin will assist users with steps 2 and 5.
 
 ## Documentation
-To learn more about how to use this plugin, see [user's documentation](https://help.cloudmergin.com/working-with-qgis-plugin.html)
+To learn more about how to use this plugin, see [user's documentation](https://merginmaps.com/docs)
```

Comment:
which page in docs?

Reply:
Not sure, the old URL was broken long time ago and as far as I can see we don't have a dedicate page for plugin anymore, all information is scattered through various places in the docs. Changed URL to page which describes installation of the plugin, hope it is ok.

Reply:
ok, let's not worry about it too much 
maybe @alex-cit can double check links are correct/make sense?